src/marqo/index.py

```diff
@@ -624,10 +623,22 @@ def _marqo_minimum_supported_version_check(self):
                 return
 
             if versioning_helpers.parse(marqo_version) < versioning_helpers.parse(min_ver):
-                mq_logger.warning(f"Your Marqo Python client requires a minimum Marqo version of "
-                                  f"{minimum_supported_marqo_version()} to function properly, but your Marqo version is {marqo_version}. "
-                                  f"Please upgrade your Marqo instance to avoid potential errors. "
-                                  f"If you have already changed your Marqo instance but still get this warning, please restart your Marqo client Python interpreter.")
+                if versioning_helpers.parse(marqo_version).major == 1:
+                    mq_logger.warning(
+                        f"Your current Marqo Python client requires a minimum Marqo version of "
+                        f"{minimum_supported_marqo_version()} to function properly. "
+                        f"Please upgrade your Marqo instance to avoid potential errors. "
+                        f"If you intended to use Marqo versions 1.x.x, "
```

Comment:
If you intended to use Marqo 1.x, please use Python client 2.x (pip install marqo==2).